pylint_odoo/test/main.py

```diff
@@ -9,6 +9,7 @@
 
 
 EXPECTED_ERRORS = 52
+EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```

Comment:
Yeah, is more easy to follow the total number of expected errors and to avoid conflicts if multiple PRs are done simultaneously.